checkIn/model.py

```diff
@@ -141,6 +141,15 @@ class TrainingVideosBridge(_base):
 	def __repr__(self):
 		return "<%s has watched videos:  %s>" % (self.user_id, self.videos_watched)
 
+	def getWatchedVideos(user):
+		db=db_session()
+		videos_query = db.query(TrainingVideosBridge).filter_by(user_id=user).one_or_none()
```

Comment:
Should type force a User object if possible so to limit possible None Type errors from the query.
```suggestion
	def getWatchedVideos(user: User):
		db=db_session()
		videos_query = db.query(TrainingVideosBridge).filter_by(user_id=user.id).one_or_none()
```